cypress/legacy-e2e-tests-guide.md

```diff
@@ -0,0 +1,107 @@
+# Legacy E2E Cypress tests
+
+> **NOTE**: As of November 2022, we are migrating away from the end-to-end, BDD-style Cypress tests in favour of writing our tests completely in JavaScript, so the following test format will eventually become obsolete.
+
+- [Running Cypress tests](#running-cypress-tests)
+- [Cypress File Structure](#cypress-file-structure)
+- [BDD Test Syntax](#bdd-test-syntax)
+
+## Running Cypress tests
+
+To locally run the tests, execute the following:
+
+1. `npm start` to start storybook server
+2. `npx cypress open --e2e` to open Cypress test runner UI
+
+## Cypress File Structure
+
+```none
+.
+├── cypress/
+│   ├── e2e/ (contains Cucumber tests as .feature files)
+│   │   ├── accessibility/
+│   │   └── common/
+│   │
+│   ├── fixtures/ (test data as json files)
+│   │
+│   ├── componentTheme/ (tests for Carbon themes)
+│   │
+│   ├── locators/
+│   │   └── [component-name]/
+│   │       ├── index.js (exported arrow functions for locators)
+│   │       └── locators.js (string const locators)
+│   │
+│   └── support/
+│       ├── step-definitions/
+│       │   └── [component-name]-steps.js (files with cucumber steps)
+│       │
+│       └── component-helper/ (contains utilities to help with component tests)
+│
+├── docker-compose.yml
+├── webpack.config.js
+├── tsconfig.json
+└── README.md
```

Comment:
Why you have removed the `cypress.config.js` file? :)

```suggestion
└── README.md
└── cypress.config.js
```

Reply:
Looks like I got my wires crossed there, thanks for spotting this @DlgSHi 👍🏼